cloud/observability/promql-to-dd-go/worker/worker.go

```diff
@@ -29,7 +30,7 @@ const (
 
 func (w *Worker) Run() {
 	interrupt := interruptCh()
-	ticker := time.NewTicker(w.QueryInterval)
+	ticker := time.NewTicker(w.SleepDuration)
```

Comment:
is this doing a 20s sleep between `[1m]` queries? I think what we want is a 60s sleep

Reply:
Yeps, that makes sense. The default value was borrowed from Python for consistency. Do you think we should make a change there too? https://github.com/temporalio/samples-server/blob/f172d775152650440ac89020272a99c380618a56/cloud/observability/promql-to-dd.py#L62

Reply:
I suppose it might just be constantly overwriting the data, which is maybe ok?